src/amo/reducers/guides.js

```diff
@@ -0,0 +1,61 @@
+/* @flow */
+import invariant from 'invariant';
+
+import { LOAD_ADDON_RESULTS } from 'core/reducers/addons';
+
+export const FETCH_GUIDES_ADDONS: 'FETCH_GUIDES_ADDONS' = 'FETCH_GUIDES_ADDONS';
+export const UPDATE_GUIDES_ADDONS_LOADED_STATUS: 'UPDATE_GUIDES_ADDONS_LOADED_STATUS' =
+  'UPDATE_GUIDES_ADDONS_LOADED_STATUS';
+
+export type GuidesState = {|
+  loading: boolean,
+|};
+
+export const initialState: GuidesState = {
+  loading: false,
+};
+
+export type FetchGuidesParams = {|
+  errorHandlerId: string,
+  guid: string,
```

Comment:
shouldn't this be `guids` since you can have multiple `guid`?

Reply:
well, I did think about that but I kept as guid since that matches the search filter name. I can change this tho..

Reply:
I ended up changing this to guids. I do think that is a better/clearer since there will most likely be multiple guids in this case.